src/components/ExportButton.tsx

```diff
@@ -86,6 +86,9 @@ export const ExportButton = ({ children, className }: ExportButtonProps) => {
         // Set the storage type.
         values.storage = storage
       }
+      if (values.type === 'ply' || values.type === 'stl') {
+        values.selection = { type: 'default_scene' }
```

Comment:
@jessfraz @alteous is hardcoding this to 'default_scene' okay for ply and stl?

Reply:
Yes that's what I'd do